README.md

```diff
@@ -99,22 +101,71 @@ then [docker-compose-no-pleuston.yml](docker-compose-no-pleuston.yml) will be us
 If you do:
 
 ```bash
-./start_ocean.sh --latest --local-parity-node
+./start_ocean.sh --latest --local-secret-store
 ```
 
-then [docker-compose-local-parity-node.yml](docker-compose-local-parity-node.yml) will be used. Read it to see what images it starts. Note that it _doesn't_ start Pleuston, and it _does_ start a Parity Secret Store.
+then [docker-compose-local-secret-store.yml](docker-compose-local-secret-store.yml) will be used. Note that it _doesn't_ start Pleuston, and it _does_ start a Parity Secret Store. The images that are started would be:
+
+- mongo:3.6
+- oceanprotocol/parity-ethereum:master (for parity client)
+- oceanprotocol/parity-ethereum:master (for secret store)
+- nginx:alpine (for secret store cluster)
+- oceanprotocol/keeper-contracts:latest
+- oceanprotocol/aquarius:latest
+- oceanprotocol/brizo:latest
 
 If you do:
 
 ```bash
-./start_ocean.sh --latest --no-pleuston --local-parity-node
+./start_ocean.sh --latest --no-pleuston --local-secret-store
+```
+
+then the last-selected Docker Compose file will be used, i.e. the one selected by `--local-secret-store`: [docker-compose-local-secret-store.yml](docker-compose-local-secret-store.yml).
+
+Finally, if you do:
+
+```bash
+./start_ocean.sh --testnet-parity-node
+```
+
+then `docker-compose-only-parity.yml` Docker Compose file will be used. It will start only one container (`oceanprotocol/parity-ethereum:master`) that will connect to the Ocean testnet network. The parity client is configure to allow connections with the RPC interface. With this option, you must place in `./parity/ocean-network/account.json` your account json and in `./parity/ocean-network/password` a file with the account passsword (default locations). Also you must export the variable `UNLOCK_ADDRESS` with the account address of this account. For example:
+
+```bash
+export UNLOCK_ADDRESS="0x00bd138abd70e2f00903268f3db08f2d25677c9e"
 ```
 
-then the last-selected Docker Compose file will be used, i.e. the one selected by `--local-parity-node`: [docker-compose-local-parity-node.yml](docker-compose-local-parity-node.yml).
+Or if you do:
+
+```bash
+./start_ocean.sh --kovan-parity-node
+```
+
+then `docker-compose-only-parity.yml` Docker Compose file will be used. It will start only one container (`oceanprotocol/parity-ethereum:master`) that will connect to the Kovan testnet network. The parity client is configure to allow connections with the RPC interface. With this option, you must place in `./parity/kovan/account.json` your account json and in `./parity/kovan/password` a file with the account passsword (default locations). Also you must export the variable `UNLOCK_ADDRESS` with the account address of this account. For example:
+
+```bash
+export UNLOCK_ADDRESS="0x00bd138abd70e2f00903268f3db08f2d25677c9e"
```

Comment:
we should use an nmemoric for this instead of private key / address / password combination